src/consensus/aft/raft.h

```diff
@@ -845,22 +858,41 @@ namespace aft
         to,
         state->last_idx);
 
+      auto progress_tracker = store->get_progress_tracker();
+      auto h = progress_tracker->get_my_hashed_nonce(
+        state->current_view, state->last_idx);
+
+      Nonce hashed_nonce;
```

Comment:
no copy ctor on Nonce?

Reply:
Nonce is currently an alias for std::array. I am trying to resist creating a new type but this might change in the near future